src/Nowruz/modules/Auth/containers/onboarding/ImageBio/useImageBio.tsx

```diff
@@ -41,6 +43,8 @@ export const useImageBio = () => {
     );
 
     delete updatedObj.cityLabel;
+    delete updatedObj.orgType;
```

Comment:
`delete` is not recommended at all in js.
and in React it is recommended to prevent mutating objects.